docs/userguide/modules/projecting-binding-data/pages/projecting-binding-data.adoc

```diff
@@ -15,6 +15,11 @@ After the backing service exposes the binding data, for a workload to access and
 
 By default, {servicebinding-title} mounts the binding data as files at a specific directory in your workload resource. You can configure the directory path using the `SERVICE_BINDING_ROOT` environment variable setup in the container where your workload runs.
 
+[NOTE]
+====
+All the projected files is read only for the user and not readable by others.
```

Comment:
which user, I suspect you mean the user that owns the files right? How do we make sure that is the user running the pod?

Reply:
This is the "container user" who runs the pod. There is no other user inside the container. I can change the word to "container user" in the docs text.